Source/Core/Core/IOS/ES/TitleContents.cpp

```diff
@@ -110,6 +110,7 @@ IPCReply ESDevice::ReadContent(u32 uid, const IOCtlVRequest& request)
   const u32 size = request.io_vectors[0].size;
   const u32 addr = request.io_vectors[0].address;
 
+  INFO_LOG_FMT(IOS_ES, "ReadContent(uid={:x}, cfd={}, size={}, addr={:08x})", uid, cfd, size, addr);
```

Comment:
Shouldn't `uid={:#x}` be used to be consistent with the previous commit? If the hex prefix isn't or shouldn't be used I think it might be a good idea to set its length `08x` like `addr` instead.

Reply:
Good catch. I chose `#x` for consistency and because UIDs typically look like `0x1xxx`.